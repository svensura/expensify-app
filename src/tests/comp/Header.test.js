import React from 'react';
import { shallow } from 'enzyme';
// import ReactShallowRenderer from 'react-test-renderer/shallow';

import Header from '../../comp/Header';

test('SHOULD RENDER Header CORRECTLY', () => {
    const wrapper = shallow(<Header />);
    // expect(wrapper.find('h1').text()).toBe('Expense App');
    expect(wrapper).toMatchSnapshot();
});

/*
test('should render Header correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Header />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});
*/