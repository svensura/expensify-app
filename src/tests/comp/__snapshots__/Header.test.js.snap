// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SHOULD RENDER Header CORRECTLY 1`] = `
<header>
  <h1>
    Expense App
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Add Expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
